python/pyspark/sql/utils.py

```diff
@@ -193,6 +193,15 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
     return cast(FuncT, wrapped)
 
 
+def require_spark_context_initialized() -> SparkContext:
```

Comment:
I would name it like: `get_jvm_spark_context`

Reply:
Good point, renamed.